Tao_Current.txt

```diff
@@ -569,63 +641,67 @@ Considering all this, it's not surprising that some IETFers would prefer to have
 
 Reporters who want to find out about "what the IETF is doing" on a particular topic would be well-advised to talk to more than one person who is active on that topic in the IETF, and should probably try to talk to the WG chair in any case. It's impossible to determine what will happen with a draft by looking at the draft or talking to the draft's author. Fortunately, all WGs have archives that a reporter can look through for recent indications about what the progress of a draft is; unfortunately, few reporters have the time or inclination to do this kind of research. Because the IETF doesn't have a press liaison, magazines or newspapers that run a story with errors won't hear directly from the IETF and therefore often won't know what they did wrong, so they might easily do it again later.
+
 9. Security Considerations
 
 Section 6.4.4 explains why each RFC is required to have a Security Considerations section and gives some idea of what it should and should not contain. Other than that information, this document does not touch on Internet security.
+
 10. Informative References
 
-[BCP9] Bradner, S., “The Internet Standards Process -- Revision 3”, BCP 9, RFC 2026, RFC 6410, October 1996.
+[BCP9](https://tools.ietf.org/html/rfc6410) Bradner, S., “The Internet Standards Process -- Revision 3”, BCP 9, RFC 2026, RFC 6410, October 1996.
 
-[BCP10] Galvin, J., “IAB and IESG Selection, Confirmation, and Recall Process: Operation of the Nominating and Recall Committees”, BCP 10, RFC 3777, June 2004.
+[BCP10](https://tools.ietf.org/html/rfc3777) Galvin, J., “IAB and IESG Selection, Confirmation, and Recall Process: Operation of the Nominating and Recall Committees”, BCP 10, RFC 3777, June 2004.
 
-[BCP11] Hovey, R. and S. Bradner, “The Organizations Involved in the IETF Standards Process”, BCP 11, RFC 2028, October 1996.
+[BCP11](https://tools.ietf.org/html/rfc2028) Hovey, R. and S. Bradner, “The Organizations Involved in the IETF Standards Process”, BCP 11, RFC 2028, October 1996.
 
-[BCP14] Bradner, S., “Key words for use in RFCs to Indicate Requirement Levels”, BCP 14, RFC 2119, March 1997.
+[BCP14](https://tools.ietf.org/html/rfc2119) Bradner, S., “Key words for use in RFCs to Indicate Requirement Levels”, BCP 14, RFC 2119, March 1997.
 
-[BCP22] Scott, G., “Guide for Internet Standards Writers”, BCP 22, RFC 2360, June 1998.
+[BCP22](https://tools.ietf.org/html/rfc2360) Scott, G., “Guide for Internet Standards Writers”, BCP 22, RFC 2360, June 1998.
 
-[BCP25] Bradner, S., “IETF Working Group Guidelines and Procedures”, BCP 25, RFC 2418, September 1998.
+[BCP25](https://tools.ietf.org/html/rfc2418) Bradner, S., “IETF Working Group Guidelines and Procedures”, BCP 25, RFC 2418, September 1998.
```

Comment:
This one is three RFCs.

I'm not going to check any further BCPs in this list, as I think it's obvious that saying "BCP XX" and linking to an RFC instead of a BCP is fraught with hazard. My recommendation is to change all links to BCP links, and to remove the RFC number from the citation itself (or, alternately, expand it to include all of the RFCs that comprise the cited BCP).

Reply:
Fixed all BCP links